themes/article/app/routes/$.tsx

```diff
@@ -47,14 +47,15 @@ export const meta: V2_MetaFunction<typeof loader> = ({ data, matches, location }
 export const links: LinksFunction = () => [KatexCSS];
 
 export const loader: LoaderFunction = async ({ params, request }) => {
-  const [first, second] = new URL(request.url).pathname.slice(1).split('/');
-  const projectName = second ? first : undefined;
-  const slug = second || first;
+  const [first, ...rest] = new URL(request.url).pathname.slice(1).split('/');
   const config = await getConfig();
-  const project = getProject(config, projectName ?? slug);
+  const project = getProject(config, first);
```

Comment:
I learned about some strangeness in the `article` theme while making these changes: the article theme does not work correctly with project slugs. I think it is ok if project slugs just do not work with `article` theme, but we should make that more clear. For example, here we should not even try to load the project based on the URL; it should always just be the default single project.

However, for this PR, I tried as best as I could to keep parity with the old functionality.